checks/raw/pinned_dependencies.go

```diff
@@ -405,10 +418,21 @@ var validateGitHubWorkflowIsFreeOfInsecureDownloads fileparser.DoWhileTrueOnFile
 
 // Check pinning of github actions in workflows.
 func collectGitHubActionsWorkflowPinning(c *checker.CheckRequest, r *checker.PinningDependenciesData) error {
-	return fileparser.OnMatchingFileContentDo(c.RepoClient, fileparser.PathMatcher{
+	err := fileparser.OnMatchingFileContentDo(c.RepoClient, fileparser.PathMatcher{
 		Pattern:       ".github/workflows/*",
 		CaseSensitive: true,
 	}, validateGitHubActionWorkflow, r)
+	if err != nil {
+		return err
+	}
+	//nolint:errcheck
+	remediationMetadata, _ := remediation.New(c)
+
+	for i := range r.Dependencies {
+		rr := &r.Dependencies[i]
+		rr.Remediation = generateRemediation(remediationMetadata, rr)
+	}
```

Comment:
same comments as above, but with `CreateWorkflowPinningRemediation` instead for 2.

Reply:
Fixed in https://github.com/ossf/scorecard/pull/3632/commits/1fcc17a47c6f05bc8095371207258c340dc46069